index.rst

```diff
@@ -98,6 +98,7 @@ The `README <https://github.com/lsst-dm/dm_dev_guide/blob/master/README.md>`__ w
    build-ci/lsstsw.rst
    build-ci/new_package.rst
    build-ci/third_party.rst
+   build-ci/github_protect_travis_flake8.rst
```

Comment:
Yeah I saw this and didn't know what the right action was. I could use the ci_overview.rst or remove it. But it should be changed. @jonathansick is out sick today but I will discuss it with him when he's back.